cli/flox/src/commands/init/mod.rs

```diff
@@ -55,6 +56,7 @@ pub struct Init {
 }
 
 impl Init {
+    #[instrument(name = "init", skip_all)]
```

Comment:
nit: i find `skip_all` being the _last_ element kinda weird, it sounds like negaring everything before (it doesn't and could be read as "skip_all[_others]" but alas).
I guess a personal taste, no worries if you disagree.

Reply:
Yeah I'd chalk that up to personal preference